src/core_plugins/kibana/public/management/sections/objects/_objects.js

```diff
@@ -153,7 +155,7 @@ uiModules.get('apps/management')
       }
 
       function refreshData() {
-        return getData($scope.advancedFilter);
+        return getData(self.advancedFilter);
```

Comment:
Feels a bit odd that the `advancedFilter` property has been changed in this PR to hang off `self` (`this`) whereas a bunch of other properties have been left to hang off `$scope`. Any reason for treating this property as special in this PR?